Mathlib/Data/Nat/Digits.lean

```diff
@@ -637,6 +637,22 @@ theorem ofDigits_mod (b k : ℕ) (L : List ℕ) : ofDigits b L % k = ofDigits (b
   ofDigits_modEq b k L
 #align nat.of_digits_mod Nat.ofDigits_mod
 
+@[simp]
+theorem ofDigits_nil (b : ℕ) : ofDigits b [] % b = 0 := rfl
```

Comment:
Isn't this also true without the `% b`?
```suggestion
theorem ofDigits_nil (b : ℕ) : ofDigits b [] = 0 := rfl
```
Also, I think this lemma could/should be placed almost directly after the defn of `ofDigits`.

Reply:
moved it, and also made argument `b` implicit.